src/layers/fp64/choropleth-layer/choropleth-layer.js

```diff
@@ -75,7 +75,6 @@ export default class ChoroplethLayer64 extends Layer {
     const IndexType = gl.getExtension('OES_element_index_uint') ?
       Uint32Array : Uint16Array;
 
-    this.setUniforms({opacity: this.props.opacity});
```

Comment:
Agreed. In addition I think the base layer applies gamma to the opacity, having it here defeats that.